patches/master_patch.patch

```diff
@@ -620,7 +620,7 @@ diff --git a/chrome/common/chrome_paths_mac.mm b/chrome/common/chrome_paths_mac.
 index d0bbbf72ff0e356e424dc280eeb1441c990b5770..8e96ea5706f87ea98ba55fcb583b16a8035fa903 100644
```

Comment:
I've been seeing these as well, but we can commit them separately to isolate any problems?